### PR TITLE
S3 Delta Integration 

### PR DESCRIPTION
**Pull request by pellicceama**
This adds support for the s3 delta integration

### CODE REVIEW

File: src/connectors/s3delta/index.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    const transformedEvent = schema.fields.reduce((acc, field) => {
+      if (field.path) {
+        acc[field.name] = this.propertyWithPath(event, field.path);
+      } else if (field.transform) {
+        acc[field.name] = field.transform(event);
+      } else {
+        acc[field.name] = null;
+      }
+      return acc;
+    }, {} as Record<string, any>);
```
Comment (Rodri77) [inline src/connectors/s3delta/index.ts:98]: is it possible to avoid any here?

File: src/connectors/s3delta/index.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    return JSON.stringify(transformedEvent);
+  }
+
+  private propertyWithPath(obj: any, path: string): any {
```
Comment (Rodri77) [inline src/connectors/s3delta/index.ts:103]: is this an object can we update the type accordingly?

**Review by Rodri77 — COMMENTED** [review]
Added some comments as questions

File: src/connectors/s3delta/index.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
+import { Connector } from '..';
```
[the remark this answers is not in the document]
Comment (pellicceama) [reply src/connectors/s3delta/index.ts:1]: yup!

File: src/connectors/s3delta/index.ts (hunk 2 above)
Comment (pellicceama) [reply src/connectors/s3delta/index.ts:103]: there'd be multiple types, so I think any is appropriate here.

**Pull request merged**